sale_update_pricelist/README.rst

```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale: Update pricelist
+======================
+
+With this module, when a pricelist is changed in the sales order, all the prices in the lines are updated with the ones defined in the pricelist.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Hugo Santos <hugo.santos@factorlibre.com>
+
+Maintainer
```

Comment:
@nhomar How can I add the runbot link?

Is this flag? [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/167/8.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-sale-workflow-167)